src/import/protobuf-c/library/protobuf-c.c

```diff
@@ -46,7 +46,7 @@
  */
 
 #include <stdlib.h>	/* for malloc, free */
-#include <string.h>	/* for strcmp, strlen, memcpy, memmove, memset */
+#include <string.h>     /* for strcmp, strlen, memcpy, memmove, memset */
```

Comment:
LAST: pls remove this extra space to have no change at all in this 3rd party code, thx

Reply:
Oopse! Fixed